packages/paste-core/components/disclosure/src/Disclosure.tsx

```diff
@@ -1,37 +1,13 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
-import {
-  useDisclosurePrimitiveState,
-  DisclosurePrimitiveInitialState,
-  DisclosurePrimitveStateReturn,
-} from '@twilio-paste/disclosure-primitive';
+import {useDisclosurePrimitiveState} from '@twilio-paste/disclosure-primitive';
 import {Box} from '@twilio-paste/box';
 import {Card} from '@twilio-paste/card';
-
-export type Variants = 'contained' | 'default';
-
-export interface DisclosureContextProps {
-  disclosure: DisclosurePrimitveStateReturn;
-  variant: Variants;
-}
-
-export const DisclosureContext = React.createContext<DisclosureContextProps>({} as any);
-
-export interface DisclosureStateReturn extends DisclosurePrimitveStateReturn {
-  [key: string]: any;
-}
-
-export type {DisclosurePrimitiveInitialState as DisclosureInitialState};
-
-export interface DisclosureProps extends DisclosurePrimitiveInitialState {
-  children: NonNullable<React.ReactNode>;
-  state?: DisclosureStateReturn;
-  variant?: Variants;
-}
+import {DisclosureContext} from './DisclosureContext';
+import {DisclosureProps, DisclosurePropTypes} from './types';
```

Comment:
`import type`

Reply:
We should lint this by enabling https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-imports.md

Reply:
I've added the rule to our eslint config. I set it to warn for now, and I'll create a ticket to do a pass of all the components and fix the imports.